src/Mod/TechDraw/Gui/QGVPage.cpp

```diff
@@ -590,7 +586,7 @@ QPixmap QGVPage::prepareCursorPixmap(const char* iconName, QPoint& hotspot)
     // therefore we must take care of the transformation ourselves...
     // Refer to QTBUG-68571 - https://bugreports.qt.io/browse/QTBUG-68571
     if (qGuiApp->platformName() == QLatin1String("xcb")) {
-        floatHotspot *= pixelRatio;
+        floatHotspot *= getDevicePixelRatio();
```

Comment:
Why do we use getDevicePixelRatio rather than the newly implemented getMaximumDPR?

Reply:
Oh right. I will replace all occurrences of these with the newly implemented function.

Reply:
This was the only place that function was being used so I removed it